src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -22,25 +23,33 @@ namespace OpenTelemetry.Exporter.Zipkin
     /// </summary>
     public sealed class ZipkinExporterOptions
     {
-        /// <summary>
-        /// Gets or sets Zipkin endpoint address. See https://zipkin.io/zipkin-api/#/default/post_spans.
-        /// Typically https://zipkin-server-name:9411/api/v2/spans.
-        /// </summary>
-        public Uri Endpoint { get; set; } = new Uri("http://localhost:9411/api/v2/spans");
+        internal const string DefaultServiceName = "OpenTelemetry Exporter";
+
+#if !NET452
+        internal const int DefaultMaxPacketSize = 4096;
+#endif
 
         /// <summary>
-        /// Gets or sets timeout in seconds.
+        /// Gets or sets the name of the service reporting telemetry.
         /// </summary>
-        public TimeSpan TimeoutSeconds { get; set; } = TimeSpan.FromSeconds(10);
```

Comment:
`TimeoutSeconds` wasn't actually being used. The ActivityProcessor now owns timeout, so I just removed it.